src/transport/IonGasTransport.cpp

```diff
@@ -45,6 +45,19 @@ void IonGasTransport::init(thermo_t* thermo, int mode, int log_level)
             m_kNeutral.push_back(k);
         }
     }
+    // set up O2/O2- collision integral [A^2]
+    // Data taken from:
+    // Prager, Jens. Modeling and simulation of charged species in
+    // lean methane-oxygen flames. Diss. 2005. Page 104.
+    vector_fp temp{300.0, 400.0, 500.0, 600.0, 800.0, 1000.0,
+                   1200.0, 1500.0, 2000.0, 2500.0, 3000.0, 4000.0};
+    vector_fp om11_O2{120.0, 107.0, 98.1, 92.1, 83.0, 77.0,
+                      72.6, 67.9, 62.7, 59.3, 56.7, 53.8};
```

Comment:
These arrays can be constant and fixed size, e.g. `const double temp[] = {300.0, ...};`